Podfile

```diff
@@ -4,4 +4,5 @@ target "nRF Toolbox" do
     pod 'EVReflection', '~> 5.10'
     pod 'SWRevealViewController', '~> 2.3'
     pod 'CorePlot', '~> 2.2'
+    pod 'Charts'
```

Comment:
The one thing the Internet hates more than the Star Wars prequels + Jar Jar Binks ...

Reply:
I know, I know. I had to fight evil and not join it...